operator/deploy/olm-catalog/ibm-spectrum-scale-csi-operator/1.1.0/ibm-spectrum-scale-csi-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -547,4 +471,5 @@ spec:
   maturity: alpha
   provider:
     name: IBM
-  version: 1.0.0
+  replaces: ibm-spectrum-scale-csi-operator.v1.0.0
```

Comment:
do we need this one, and should it be v1.0.1? since the v1.1.0 replaces that one.

Reply:
And I noticed that 
```
$ ls -ltr
total 8
drwxr-xr-x  4 vhu  staff  128 Feb 19 10:06 1.0.1
drwxr-xr-x  4 vhu  staff  128 Feb 19 10:06 1.1.0
```

should we be keeping the 1.0.1 directory?  it seems like the 1.1.0 replaced 1.0.0,.. or should we keep the last 2? What's the thought?

Reply:
I removed 1.0.0 because we needed to change the name of the crd and cr and it was causing testing failures.

@ replaces: since we're not releasing v1.0.1 to Operatorhub, this is the correct transition.